### ISSUE TITLE
High & Dark Elf lord/hero monstrous mounts having Prowess/Mastery traits (except Rakarth).

### ISSUE THREAD

**Issue by Bogdanov89** [chadvandy/cbfm_wh3#1231]
**Describe the bug**
**EDIT: CA has made their stance on this officially known, their reply is at the bottom of this thread.
All monstrous mounts for HE & DE lords/heroes must lose prowess/mastery traits , except for Rakarth.**

Some HElf & DElf characters are missing their racial traits Martial Prowess/Mastery or Murderous Prowess/Mastery when placed on certain mounts.
High Elf mounts that strip their Elf rider of the elf racial traits: All griffons, All dragons, Teclis arcane phoenix, Alarielle great eagle (other great eagles do not remove the traits), Eltharions stormwing griffon (its called Grim Discipline for Eltharion).
Dark elf mounts that strip their Elf rider of the elf racial traits: black dragons (except Rakarths Bracchus black dragon).

**Expected behavior**
Mounts should not remove/change their riders racial traits.
For example Settra does not get the Construct trait when on the gigantic Warspinx, and the living Aranessa does not gain the Undead trait when mounted on the undead crab.
Wulfrik has the usual norscan Rage trait which he keeps even when on the War Mammoth mount.
And the Mortal Chaos humans from Khorne do not lose their regular mortal/living leadership when mounted on a Demonic beast.
Also the size of the mount does not matter - for example Manticores all properly keep their elf riders racial traits despite being larger than griffons and eagles.

**To Reproduce**
1. Check HE & DE traits in custom battle

**Additional context**
The consistency between lords/heroes and their mounted versions has always been lacking in TWW game series.
For example lords/heroes would lose Encourage when mounted - or they would lack the Siege Attacker attribute despite their monstrous mount having it on its own (as a unit).
CA also does not adhere to much "realism" when it comes to these trait transfers - for example Skink Chief has traits for Vanguard deployment & Stalk and they both work even on the gigantic Stegadon mount.

**Comment by allenhu0912** [comment]
I don't think this is a bug. The idea being that the mount is doing most of the fighting rather than the rider. Unmounted dragons don't have racial traits, for example, so they don't have it either when they are mounted. Rakarth on his dragon does get the trait because he can give monsters in his army Murderous Prowess. Mounted Manticores get the trait because they are literally living embodiments of Khaine, etc.

**Comment by Bogdanov89** [comment]
Sorry but as examples given above show in the vast majority of cases the Rider traits are the dominant ones.
Even generic lord/hero traits like Strider will apply to any mount the lord/hero uses.

If your logic was true then Lords/heroes would get traits like Undead, Demonic, Construct etc just because they mounted a creature with that trait - which is obviously false.

There is great inconsistency in the HE & DE characters when it comes to racial traits, but outside of HE & DE there does not seem to be this problem.

**Comment by Bogdanov89** [comment]
I mean this is the distance at which my gigantic Stegodon (Skink CHief mount) got revealed, and that is after using vanguard deployment.
I dont think realism & logic apply to the case of Lord traits working on mounts:

![image](https://github.com/chadvandy/cbfm_wh3/assets/55892027/707b18e2-f5a2-49ef-b18c-9b3c22eaf9ad)

**Comment by allenhu0912** [comment]
> Sorry but as examples given above show in the vast majority of cases the Rider traits are the dominant ones. Even generic lord/hero traits like Strider will apply to any mount the lord/hero uses.
> 
> If your logic was true then Lords/heroes would get traits like Undead, Demonic, Construct etc just because they mounted a creature with that trait - which is obviously false.
> 
> There is great inconsistency in the HE & DE characters when it comes to racial traits, but outside of HE & DE there does not seem to be this problem.

Strider, Undead, Demonic, Construct etc. are not _racial_ traits. Characters on dragons don't get racial traits like Murderous Prowess because the dragon is doing the actual fighting (i.e. the stats are highly similar to the mount, rather than the original character) and a Black Dragon is literally not a Dark Elf. Why should a dragon get the Dark Elf racial trait when it is not a Dark Elf?

**Comment by Bogdanov89** [comment]
> > Sorry but as examples given above show in the vast majority of cases the Rider traits are the dominant ones. Even generic lord/hero traits like Strider will apply to any mount the lord/hero uses.
> > If your logic was true then Lords/heroes would get traits like Undead, Demonic, Construct etc just because they mounted a creature with that trait - which is obviously false.
> > There is great inconsistency in the HE & DE characters when it comes to racial traits, but outside of HE & DE there does not seem to be this problem.
> 
> Strider, Undead, Demonic, Construct etc. are not _racial_ traits. Characters on dragons don't get racial traits like Murderous Prowess because the dragon is doing the actual fighting (i.e. the stats are highly similar to the mount, rather than the original character) and a Black Dragon is literally not a Dark Elf. Why should a dragon get the Dark Elf racial trait when it is not a Dark Elf?

> > Sorry but as examples given above show in the vast majority of cases the Rider traits are the dominant ones. Even generic lord/hero traits like Strider will apply to any mount the lord/hero uses.
> > If your logic was true then Lords/heroes would get traits like Undead, Demonic, Construct etc just because they mounted a creature with that trait - which is obviously false.
> > There is great inconsistency in the HE & DE characters when it comes to racial traits, but outside of HE & DE there does not seem to be this problem.
> 
> Strider, Undead, Demonic, Construct etc. are not _racial_ traits. Characters on dragons don't get racial traits like Murderous Prowess because the dragon is doing the actual fighting (i.e. the stats are highly similar to the mount, rather than the original character) and a Black Dragon is literally not a Dark Elf. Why should a dragon get the Dark Elf racial trait when it is not a Dark Elf?

At this point i believe you are trolling.

**Did you miss the part where the Great Eagles mounts have racial martial prowess, or where the War Mammoths mounts have Norsca human Rage racial passive?**
Or where literally any attack modifier (poison, sunder armor etc) that the lord/hero has will also somehow work on the mounts attacks?
Undead, Demonic, Constructs & "living" are racial traits - this game has nothing closer than that to them.

Again, who is visually doing the attacks has nothing to do with this.
This is about consistency in how traits are applied when the lords & heroes change mounts.
There are literally hundreds of characters in TWW 3 and the general rule so far has been that Lords racial traits are applied regardless of what mount is used.

**Please stop with this nonsense about "realism" of who is doing the attack animations when its already clear that this game has tons of examples where that rule is utterly broken and irrelevant.**

**Comment by Harmandodelavega** [comment]
lol

**Comment by Harmandodelavega** [comment]
I'm not reading that

**Comment by Drazhir2** [comment]
answer from CA on Bogdanov's total war forum post https://forums.totalwar.com/discussion/333829/helf-delf-lords-missing-racial-traits-when-on-certain-mounts
![image](https://github.com/chadvandy/cbfm_wh3/assets/135981276/6aa5bca3-f7c6-4ff3-8d6f-05d396f163e7)

**Comment by Bogdanov89** [comment]
Seems you were faster than me in posting it here :)
Thank you!

**Comment by Harmandodelavega** [comment]
Bogdanov we don't care what must be done, go bother CA

**Comment by mfvreeland** [comment]
Going to let CA handle any changes required (or not required) here.
